main/syslog-ng/APKBUILD

```diff
@@ -1,18 +1,25 @@
 # Contributor: jv <jens@eisfair.org>
 # Contributor: Adrian Guenter <adrian@gntr.me>
 # Contributor: Jakub Jirutka <jakub@jirutka.cz>
+# Contributor: Michael Pirogov <vbnet.ru@gmail.com>
 # Maintainer: jv <jens@eisfair.org>
 pkgname=syslog-ng
 pkgver=3.22.1
-pkgrel=1
+pkgrel=2
 pkgdesc="Next generation logging daemon"
 url="http://www.balabit.com"
 arch="all"
 license="GPL-2.0-or-later"
+options="!check" # no criterion on alpine
```

Comment:
Not quite sure if I understand this.

Reply:
http://www.serpentine.com/criterion needed for the test

Reply:
Maybe rephrase it to  "Missing dep: criterion" then to make it clearer what you mean.

Reply:
done. Thank you:)